hardhat.config.ts

```diff
@@ -70,16 +70,18 @@ const config: HardhatUserConfig = {
       gasMultiplier: 10,
       timeout: 12000000,
     },
-    hardhat: (() => {
+    hardhat:
+     (() => {
       if (process.env.BSC_ARCHIVE_NODE) {
         return {
           chainId: 56,
           forking: {
             url: process.env.BSC_ARCHIVE_NODE || '',
-          }
+          },
+          allowUnlimitedContractSize: true
```

Comment:
Does it work without unlimited contract size now?

```suggestion
          }
```

Reply:
Yeah it works, I have removed it in next commit